ui/src/menu.tsx

```diff
@@ -51,6 +53,7 @@ export const XMenu = ({ model }: { model: Menu }) => {
     <div data-test={name} className={clas(css.card, 'w-menu')} ref={ref} onClick={toggleMenu}>
       {image && <Fluent.Persona imageUrl={image} size={Fluent.PersonaSize.size48} styles={{ details: { padding: 0 } }} />}
       {icon && <Fluent.FontIcon className={css.icon} iconName={icon} />}
+      {label}
```

Comment:
Let's add a bit of styling and themability.

```suggestion
      {title && <Fluent.Text variant='mediumPlus' styles={{ root: { color: cssVar('$text') } }}>{title}</Fluent.Text>}
```